{{cookiecutter.repo_name}}/requirements_dev.txt

```diff
@@ -1,2 +1,4 @@
 bumpversion==0.5.3
 wheel==0.23.0
+cryptography==1.0.1
+PyYAML==3.11
```

Comment:
Let's wrap these 2 lines in `{% if cookiecutter.use_pypi_deployment_with_travis == 'y' -%}`